android/uhabits-core/src/test/java/org/isoron/uhabits/core/models/HabitTest.java

```diff
@@ -102,19 +102,19 @@ public void test_isCompleted_numerical() throws Exception
         h.setTargetValue(100.0);
         assertFalse(h.isCompletedToday());
 
-        h.getRepetitions().toggle(getToday(), 200);
+        h.getRepetitions().toggle(getToday(), 200 * 1000);
```

Comment:
I'm a bit at odds with changing unit tests. It seems the app stores the numerical value as an int with a set multiplier.
Now I also don't want to go around everywhere changing the way numerical values are saved. I feel like that may require a bit more coordination.